dockerfiles/sv-pipeline-base/Dockerfile

```diff
@@ -1,19 +1,19 @@
 # This is the base dockerfile for the GATK SV pipeline that adds dependencies
 # for the sv-pipeline, sv-pipeline-qc, and sv-pipeline-rdtest
+ARG SVBASE_IMAGE=sv-base:latest
+ARG CONDABASE_IMAGE=conda-base:latest
```

Comment:
No, these values are always overridden by the action of build_docker.py. It is the code style of the Dockerfiles in the project that they have these default values (presumably in case someone wanted to try their hand at building these images by hand?). You can check out any of the other Dockerfiles and see that they're all this way. But the Dockerfiles could be changed to have the default values omitted and they would still work with build_docker.py.

Reply:
That makes sense, thanks for the clarification!